dot/parachain/dispute/test_helpers.go

```diff
@@ -108,3 +127,31 @@ func activateLeaf(
 	participation.ProcessActiveLeavesUpdate(update)
 	return nil
 }
+
+func GetBlockNumberHash(blockNumber parachainTypes.BlockNumber) common.Hash {
+	encodedBlockNumber, err := scale.Marshal(blockNumber)
+	if err != nil {
+		panic("failed to encode block number:" + err.Error())
+	}
+
+	blockHash, err := common.Blake2bHash(encodedBlockNumber)
+	if err != nil {
+		panic("failed to hash block number:" + err.Error())
+	}
+
+	return blockHash
+}
+
+func DummyActivatedLeaf(blockNumber parachainTypes.BlockNumber) overseer.ActivatedLeaf {
+	return overseer.ActivatedLeaf{
+		Hash:   GetBlockNumberHash(blockNumber),
+		Number: uint32(blockNumber),
+	}
+}
+
+func NextLeaf(chain *[]common.Hash) overseer.ActivatedLeaf {
```

Comment:
Yeah, I've been meaning to revert this. Lemme see what I can do!

Reply:
I remember now; this is needed for mocks. The `chain` undergoes runtime changes during the tests and those changes are needed by the mocks (overseer and runtime methods). 

We can't make it global since the tests run in parallel. Can you think of a better approach for this?

Reply:
I don't have anything specific, but since it is used for testing, I would suggest passing the `t *testing.T` and call the `t.Helper()`